junit-jupiter-engine/src/test/java/org/junit/jupiter/api/extension/ExtensionComposabilityTests.java

```diff
@@ -64,7 +64,7 @@ void ensureJupiterExtensionApisAreComposable() {
 
 		// 3) Dynamically implement all Extension APIs
 		Object dynamicKitchenSinkExtension = Proxy.newProxyInstance(getClass().getClassLoader(),
-			extensionApis.toArray(new Class<?>[extensionApis.size()]), (proxy, method, args) -> null);
+			extensionApis.toArray(new Class<?>[0]), (proxy, method, args) -> null);
```

Comment:
You could push this to `master` as a separate polishing commit.

Reply:
Will revert all changes made to `ExtensionComposabilityTests.java` in this PR.